dotnet/samples/KernelSyntaxExamples/Example07_BingAndGoogleSkills.cs

```diff
@@ -77,7 +77,7 @@ private static async Task Example1Async(IKernel kernel, string searchSkillId)
 
         // Run
         var question = "What's the largest building in the world?";
-        var result = await kernel.Skills.GetFunction(searchSkillId, "search").InvokeAsync(question);
+        var result = await kernel.Skills.GetFunction(searchSkillId, "search").InvokeAsync(question, kernel);
```

Comment:
```suggestion
        var func = kernel.Skills.GetFunction(searchSkillId, "search");
        var result = await kernel.RunAsync(func, new() { ["input"] = question });
```

Reply:
Used `kernel.RunAsync(question, func);` signature, which will be an extension in the future.